test/DynamoCoreTests/UnitsOfMeasureTests.cs

```diff
@@ -779,7 +779,11 @@ public void CanCreateForgeUnitType_FromFutureTypeString()
             var unitType = Unit.ByTypeID($"{milimeters}-99.9.9");
             Assert.NotNull(unitType);
             Assert.AreEqual("Millimeters", unitType.Name);
-            Assert.AreEqual($"{milimeters}-2.0.0", unitType.TypeId);
+            
+            // Depending on the test system, the latest known version could be either -2.0.0 or -1.0.1
+            var expectedVersions = new[] { $"{milimeters}-2.0.0", $"{milimeters}-1.0.1" };
+            Assert.That(expectedVersions, Contains.Item(unitType.TypeId), 
+                $"Expected TypeId to be one of the known versions, but got: {unitType.TypeId}");
```

Comment:
Depending on whether ASC schemas are available on the test system, this may be either `1.0.1` or `2.0.0`. The schema fallback mechanism correctly selects the latest available schema when encountering an unknown future version, but the specific version depends on the current system configuration.